docs/source/api/traitsui.editors.led_editor.rst

```diff
@@ -0,0 +1,7 @@
+traitsui\.editors\.led\_editor module
+=====================================
+
+.. automodule:: traitsui.editors.led_editor
+    :members:
+    :undoc-members:
+    :show-inheritance:
```

Comment:
The API documentation is now being generated automatically. So this file and `traitsui.editors.rst` are not needed any more.